package.json

```diff
@@ -83,6 +83,9 @@
     "typescript": "^5.4.5",
     "yorkie": "^2.0.0"
   },
+  "peerDependencies": {
+    "natural-compare": "*"
```

Comment:
I hate to add a dependency, but since `natural-compare` is already a dependency of ESLint then I hope this is OK.

Reply:
It's fine to use this package. Let's move in to `dependencies` instead of `peerDependencies`.

Reply:
@nzakas Should it be pegged to a specific version? I used `peerDependencies` and no specific version since `natural-compare` is a dependency of `eslint` [here](https://github.com/eslint/eslint/blob/main/package.json#L132), and `eslint` will presumably always be installed alongside `eslint/json`, so presumably we would want to defer to whatever version of `natural-compare` is required by `eslint`?

Although I notice `eslint` is a `devDependency` of `eslint/json` and not a `dependency` [here](https://github.com/eslint/json/blob/main/package.json#L74), so maybe that's not correct...?

Reply:
Added more tests for nesting: https://github.com/eslint/json/pull/76/commits/3c1270a774dae4eb79f29e38cc113db9d15c71e4

Reply:
I also think this should be added to `dependencies`.

```diff
   "dependencies": {
     "@eslint/plugin-kit": "^0.2.3",
     "@humanwhocodes/momoa": "^3.3.4",
+    "natural-compare": "^1.4.0"
   },
```

Reply:
Done! https://github.com/eslint/json/pull/76/commits/a1328dfed2f57e56c97d19ae6474114e6508da67